### ISSUE TITLE
Add a "skip to content" link

### ISSUE THREAD

**Issue by ggeisler** [projectblacklight/arclight#550]
Since there are a number of links and other focusable elements on every ArcLight page that appear before the main content of the page, we should consider adding a skip-to-main-content link. This will enable a screenreader or keyboard-only user to more get to the main page content more quickly.

If we add a `main` landmark element as suggested in #548 this becomes somewhat less critical, since the user could navigate to the `main` landmark more efficiently than she can currently, but a skip link would be an even more efficient means to get to the main content of a page.

**Comment by ggeisler** [comment]
Superseded by #840.
